packages/11ty/_includes/components/figure/image/html.js

```diff
@@ -10,49 +10,33 @@ const path = require('path')
  */
 module.exports = function(eleventyConfig) {
   const annotationsUI = eleventyConfig.getFilter('annotationsUI')
-  const canvasPanel = eleventyConfig.getFilter('canvasPanel')
   const figurecaption = eleventyConfig.getFilter('figurecaption')
+  const figureimageelement = eleventyConfig.getFilter('figureimageelement')
```

Comment:
Should these names be camel case?
```suggestion
  const figureImageElement = eleventyConfig.getFilter('figureImageElement')
```

Reply:
yep, I was following the naming of the rest of the figures, I can rename this one and then open a subsequent PR for renaming other figure elements / stripping semicolons from web components